portfolio/src/main/webapp/map-constants.js

```diff
@@ -0,0 +1,127 @@
+/** 
+ * A custom color scheme of the map + basic properties for initialization
+ * (like initial center of map + zoom level).
+ */
+const mapOptions = {
```

Comment:
If these are constants, they should be capital, right? So this should be MAP_OPTIONS

Reply:
YES